examples/demo_dlmbl/solution.py

```diff
@@ -905,28 +1273,95 @@ def min_max_scale(input):
     logger=TensorBoardLogger(
         save_dir=log_dir,
         name="phase2fluor",
-        version="wider",
+        version="phase2fluor_low_lr",
         log_graph=True,
     ),
     fast_dev_run=True,
 )  # Set fast_dev_run to False to train the model.
-trainer.fit(phase2fluor_wider_model, datamodule=phase2fluor_data)
+trainer.fit(phase2fluor_model_low_lr, datamodule=phase2fluor_2D_data)
+# %% [markdown]
+"""
+<div class="alert alert-info">
+<h3>
+Extra Example 2: 3D Virtual Staining
+</h3>
+Now, let's implement a 3D virtual staining model(Phase->Fluorescence)<br>
+<b>Note:</b> This task might take longer to train +1 hr. Try it out in your free-time.
+
+</div>
+"""
+
+# %% tags=["task"]
+data_path = Path()  # TODO: Point to a 3D dataset (HEK, Neuromast)
+BATCH_SIZE = 4
+YX_PATCH_SIZE = (256, 256)
+
+phase2fluor_3D_config = ...
+
+phase2fluor_3D_data = HCSDataModule(...)
+
+phase2fluor_3D = VSUNet(...)
+
+trainer = VSTrainer(...)
+
+# Start the training
+trainer.fit(...)
 
 # %% tags=["solution"]
 
 ##########################
 ######## Solution ########
 ##########################
+"""
```

Comment:
@edyoshikun and @ziw-liu have you executed this and the following cells to test that the model can be trained? If so, let's keep the VSCyto3D model here. Otherwise, you can delete this part of the exercise.